backend/src/errors/validationError.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * This file contains the class of errors pertaining
+ * to validation errors
+ */
+
+import { Error } from "./errors";
+
+/**
+ * This class is used to contain any error messages pertaining
+ * to validating inputs.
+ */
+const INVALID_EMAIL_ID = "Invalid email was found, email must not have @iusd.org.";
```

Comment:
An issue I faced with this approach in LAKTAA was that as we developed more, we had a lot of error messages so maintaining all of them like this became a pain. Maybe something to note for future